### PR TITLE
Borg Adjustments

### PR DESCRIPTION
**Pull request by LadyFowls**
- balance: "Medical borg's are now able to handle tanks, breath mask's and organs with chem gripper."
  - tweak: "Renames Hunter Killers to Military Frames."

### CODE REVIEW

**Comment by dragoslava** [comment]
!review

File: code/modules/organs/subtypes/machine.dm [hunk 1]
```diff
@@ -77,6 +77,9 @@
 		emp_counter--
 
 /obj/item/organ/cell/emp_act(severity)
+	if(owner.get_species() == "Military Frame")
```
Comment (Sindorman) [inline code/modules/organs/subtypes/machine.dm]: After discussing this in discord. Arrow, Paradox and others agree that HK should use surge module(an organ). But it should regenerate over time

File: html/changelogs/hk47.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - balance: "Medical borg's are now able to handle tanks, breath mask's and organs with a new gripper."
+  - tweak: "Renames Hunter Killers to Military Frames."
```
Comment (Geevies) [inline html/changelogs/hk47.yml]: Might be good to add here that medborgs are now losing their medical tool suite, and should now use normal surgical tools found in ORs. Gonna leave this as a comment only, cuz it's not explicitly needed.

Comment (Sindorman) [reply html/changelogs/hk47.yml]: Agreed!

Comment (LadyFowls) [reply html/changelogs/hk47.yml]: Fixed

**Comment by Arrow768** [comment]
With the removal of the surgical tools from the medical borg,
what are its differences to the emergency cyborg?

**Comment by LadyFowls** [comment]
gonna adjust some stuff

**Pull request merged**